agent/cmd/serve.go

```diff
@@ -218,4 +220,5 @@ func init() {
 	serveCmd.Flags().StringArrayVar(&_audiences, "kube-audience", nil, "The audience to expect in the token (for kubernetes auth mode)")
 	serveCmd.Flags().StringVar(&_workspaceNamespace, "kube-workspace-namespace", os.Getenv("WORKSPACE_NAMESPACE"), "The Workspace object namespace (for kubernetes auth mode)")
 	serveCmd.Flags().StringVar(&_workspaceName, "kube-workspace-name", os.Getenv("WORKSPACE_NAME"), "The Workspace object name (for kubernetes auth mode)")
+	serveCmd.Flags().UintVar(&_pulumiLogLevel, "pulumi-log-level", 0, "The level of logging to use for the Pulumi CLI")
```

Comment:
Seems the default should be `1` to match the default log level in Pulumi.  I know it works with zero but one seems clearer to me.  WDYT?

Reply:
The default if `--verbose` is not provided is `0` [[ref](https://github.com/pulumi/pulumi/blob/425a5d703969ee9218b56e655b9085bc86909d1b/pkg/cmd/pulumi/pulumi.go#L331-L332)].

It differentiates between whether verbose logging was not enabled or not: https://github.com/pulumi/pulumi/blob/master/sdk/go/common/util/logging/log.go#L106. This suggests that there is a difference between `0` and `1`.